test/CVXStrategy.js

```diff
@@ -167,7 +174,7 @@ describe("CVXStrategy", function () {
 
         expect(Number(await strategy.ethToWant(oneUnit))).to.be.greaterThan(0);
         expect(Number(await strategy.crvToWant(oneUnit))).to.be.greaterThan(0);
-        expect(Number(await strategy.cvxToWant(oneUnit))).to.be.greaterThan(0);
+        // expect(Number(await strategy.cvxToWant(oneUnit))).to.be.greaterThan(0);
```

Comment:
Suggestions: do not commit unelaborated comments, it makes code review even harder.